CHANGELOG-3.3.md

```diff
@@ -28,6 +28,10 @@ Note that any `etcd_debugging_*` metrics are experimental and subject to change.
 
 - Upgrade [`github.com/coreos/bbolt`](https://github.com/etcd-io/bbolt/releases) from [**`v1.3.1-coreos.6`**](https://github.com/etcd-io/bbolt/releases/tag/v1.3.1-coreos.6) to [**`v1.3.3`**](https://github.com/etcd-io/bbolt/releases/tag/v1.3.3).
 
+### Package `etcdctl`
```

Comment:
Let's remove `Package`?

Reply:
Also, just a thought for small change along with @Gyuho 's comment. How about re-writing line below to "Fix `etcdctl member add` (https://github.com/etcd-io/etcd/pull/11194) command to prevent potential timeout." We use "fix" vs "fixed" in general. Thanks!

Reply:
sounds good. Thanks!

Reply:
Thanks @jingyih